### PR TITLE
Finished NodeJS week3 homework

### PR DESCRIPTION
**Pull request by RodrigoGuinsbergPinto**

### CODE REVIEW

File: src/backend/api/meals.js [hunk 1]
```diff
@@ -5,8 +5,96 @@ const knex = require("../database");
...
+    let query = knex("meal");
+
+    if ("maxPrice" in req.query) {
+      const maxPrice = parseInt(req.query.maxPrice);
+      if (!isNaN(maxPrice)) query = query.where("price", "<", maxPrice);
+      else {
+        return res
+          .status(404)
+          .send(`Error: 💥 Please provide a maxprice for the meal.`);
+      }
```
Comment (r0hitsharma) [inline src/backend/api/meals.js:17]: Should invalid input be 404 or another code?

File: src/backend/api/reviews.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    if (meal.length === 0) {
+      return res.status(404).send(`Meal id doesn't exist.`);
+    }
+    const updatedReview = await knex("review").where({ id: reviewId }).update({
+      description: req.body.description,
+      title: req.body.title,
+      meal_id: req.body.meal_id,
+      stars: req.body.stars,
+      created_date: req.body.created_date,
+    });
```
Comment (r0hitsharma) [inline src/backend/api/reviews.js:72]: Would this still work if not all fields are provided?

**Review by r0hitsharma — APPROVED** [review]
Great work :) Minor comments.

File: src/backend/api/reviews.js (hunk 2 above)
Comment (RodrigoGuinsbergPinto) [reply src/backend/api/reviews.js:72]: Thank you for the comments. I will try to implement the code.

**Pull request merged**